docs/_documentations/troubleshooting.md

```diff
@@ -119,9 +119,30 @@ When attempting to install Codewind in IntelliJ, you cannot locate the Codewind
 
 This error occurs because the custom plug-in repository link contains an additional space.
 
-**Workaround**
+**Workaround:**
 Remove the extra space from the custom plug-in repository link.
 
+<!--
+Action/Topic: Installing Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/2563
+0.11.0: New
+-->
+## Without a keychain or keyring, local Codewind crashes when certain commands are used
+Local Codewind needs to use a keychain, also called a keyring, when certain commands are used.
+
+**Workaround:**
+Use any keyring compatible with `go-keyring`, which is what Codewind uses internally. These compatible keychains are included with the following operating systems:
+- MacOS has Keychain.
```

Comment:
I don't think it really helps to list these; you can just say that if you're on mac or windows 10 you already have a keychain.